apps/consulting/posts/categories.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "name": "Data Visualization",
+    "value": "data-visualization"
```

Comment:
@bskinn I can change it to `slug`, but that's what the Storyblok datasources categories schema looks like z`name` + `value`. So I wanted it to match in both places. I just thought it might be confusing at first with the eventual debugging if someone display fetched Storyblok categories with `name` + `value` schema and local ones with `name` + `slug`. And then they are merge together so It's just easier to make schema in botch places the same. But as I said if you like it to have `slug` in local categories then it's easy fix.

Reply:
Ah, yes, keeping it consistent with the Storyblok interface is a good idea. 

No change here, stick with `name` & `value`. Thanks!